pages/services/confluent-zookeeper/2.3.0-4.0.0e/release-notes/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Release Notes
+menuWeight: 120
+model: /services/kafka-zookeeper/data.yml
+render: mustache
+---
+# Version 2.3.0-4.0.0e
+
+## Features
+- All frameworks (Confluent ZooKeeper included) now isolate their `/tmp` task directories by making them Mesos [`SANDBOX_PATH` volume sources](https://github.com/apache/mesos/blob/master/docs/container-volume.md#sandbox_path-volume-source). ([#2467](https://github.com/mesosphere/dcos-commons/pull/2467) and [#2486](https://github.com/mesosphere/dcos-commons/pull/2486))
```

Comment:
`s/Confluent ZooKeeper/{{ model.techName }}/`?

Reply:
Then I have to edit it again when I copy this text to the github release notes 😢

Reply:
Ah, it's fine then.